meteor/client/lib/rundownTiming.ts

```diff
@@ -732,6 +769,10 @@ export interface RundownTimingContext {
 	breakIsLastRundown?: boolean
 	/** Was this time context calculated during a high-resolution tick */
 	isLowResolution: boolean
+	/** The absolute start time of the next break, if any. */
+	nextBreakStartTime?: number
+	/** The absolute end time of the next break, if any. */
+	nextBreakEndTime?: number
```

Comment:
Does it make any sense for the calculator to expose both of these? Maybe it would make more sense to expose a single `nextRundownAnchror`?